pylintrc_backup

```diff
@@ -142,6 +142,7 @@ enable =
 	no-self-argument,
```

Comment:
we don't need to push pylintrc_backup file. It is only generated for local use and a backup option when running edx_lint write pylintrc.